charts/longhorn/v1.1.3/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: longhorn
-version: 1.1.2
-appVersion: v1.1.2
-kubeVersion: ">=v1.16.0-r0"
+version: 1.1.3
+appVersion: v1.1.3
+kubeVersion: ">= v1.16.0-0, < v1.22.0-0"
```

Comment:
@cbron @brendarearden we don't need annotations here right? Since this chart repo is meant for users in earlier versions of Rancher 2.4 / 2.5, who will not be getting the kubeVersion update.

Reply:
I guess if someone wanted to do a vanilla helm install of 1.1.3, we wouldn't want them to install on a 1.22 cluster, so I don't see a big problem with including it as a hard stop.

Reply:
Yes, this is one purpose. cc @PhanLe1010 @khushboo-rancher